workspaces/src/network/account.rs

```diff
@@ -174,59 +187,60 @@ impl Contract {
     ) -> anyhow::Result<CallExecutionDetails> {
         self.account.delete_account(worker, beneficiary_id).await
     }
+
+    /// Start a batch transaction, using the current contract as the signer and
+    /// making calls into this contract. Returns a [`Transaction`] object that
+    /// we can use to add Actions to the batched transaction. Call `transact`
+    /// to send the batched transaction to the network.
+    pub fn batch_tx<'a, T: Network>(&self, worker: &'a Worker<T>) -> Transaction<'a> {
```

Comment:
I think I would prefer this function to be called `call_batch` to stay consistent with other call-"initiating" functions. We will have to call `.transact()` in the end anyway, so the transactional nature is implied here.

Reply:
actually this might not be a great idea for naming either calls are more associated to making calls into contract functions, which doesn't suit all cases this batch transaction is meant for such as staking or account creation:
```rust
account.call_batch(new_account_id)
    .create_account()
    .transfer(...)
    .stake(...)
```

Reply:
Good point, yeah. In that case, I would go for either your original option or for a simple `batch`. Regardless, I do not really have a strong opinion here.

Reply:
Renamed to `batch` instead since the `tx` is kinda redundant at the end of the day